src/sssom/parsers.py

```diff
@@ -977,6 +978,27 @@ def split_dataframe(
     )
 
 
+@dataclass(frozen=True, eq=True)
+class SSSOMSplitTriple:
+    subject_prefix: str
+    object_prefix: str
+    relation: str
+
+    def __post_init__(self):
+        relation_prefix, relation_id = self.relation.split(":")
```

Comment:
use the `curies` package for CURIE parsing! Within the MSDF, you can access the converter directly with `msdf.converter` and use `msdf.converter.parse_curie`